src/observers/export.vars.files.csv/README.md

```diff
@@ -1,13 +1 @@
 # export.vars.files.csv observer
-
```

Comment:
I get the idea of keeping the readme simple and use dedicated structure for doc, but what is the benefit of moving the whole body of the readme.md elsewhere since it is the conventional "entry point" when looking a repo? Maybe a tradeoff to find, and content redundancy resolution between the three non-equivalent locations:
- ware description in json
- readme.md (not mandatory but usually good practice)
- doc content in dedicated folder